templates/_layouts/html.twig

```diff
@@ -69,6 +69,21 @@
       
       {% block page %}{% endblock %}
       
+      {% block script_inline %}
+        {# JS Constants #}
+        {% set jsConstants = {
+          photoswipe: {
+            lightbox: 'photoswipe-lightbox.min.js',
+            module: 'photoswipe.min.js',
+          }|map((href, key) => alias('@assetBaseUrl/build/' ~ href|replace('/.(css|js)/', ".#{assetManifest[href]}.$1"))),
+        }|merge(jsConstants ?? {})|filter %}
+        <div id="pjax-eval">
```

Comment:
Any significance of the `id="pjax-eval"`? Do we have to add this to the pjax selectors list?

Reply:
Added to pjax selectors list after checking the same usage in _Vahura_ [code](https://github.com/miranj/vahura/blob/a1e25c873512c967f57c6b87af33d960a6f26fd2/assets/js/pjaxinit.js#L14).